File: Upcoder/Khối 11/Chương 1 Làm Quen/Tinh Cos/TKN_K11_Cos.py
import math
x = float(input())
tong = ((abs(2 * x +3 )/ math.cos(x))+ math.sqrt(2 + math.cos(x)))
tong = round(tong, 2) # làm tròn 2 chữ số thập phân
print(tong,end ="")

File: Upcoder/Khối 11/Chương 1 Làm Quen/Tinh Cos/TKN_K11_Cos.cs
using System;
namespace Tinhcos
{
    public class Program
    {
        static void Main(string[]args)
        {
            double x = double.Parse(Console.ReadLine());
            double tong = ((Math.Abs(2 * x + 3) / Math.Cos(x) ) + Math.Sqrt(2 + Math.Cos(x)));
            Console.Write("{0:0.00}",tong); // Làm tròn 2 só thập phân
        }
    }
}